tpm/tpm_test.go

```diff
@@ -604,3 +606,33 @@ func TestForceClear(t *testing.T) {
 		t.Fatal("Couldn't clear the TPM without owner auth in physical presence mode:", err)
 	}
 }
+
+func TestNVWriteValue(t *testing.T) {
+	mathrand.Seed(time.Now().Unix())
```

Comment:
Nit: Its usually good to keep tests as deterministic as possible. As such, consider using a fixed seed: `mathrand.Seed(42)`